lms/envs/devstack_docker.py

```diff
@@ -44,7 +44,7 @@
 })
 
 ENABLE_MKTG_SITE = os.environ.get('ENABLE_MARKETING_SITE', False)
-MARKETING_SITE_ROOT = os.environ.get('MARKETING_SITE_ROOT', 'http://wordpress.edx.devstack.lms:8888/')
+MARKETING_SITE_ROOT = os.environ.get('MARKETING_SITE_ROOT', 'http://wordpress.edx.devstack.lms/')
```

Comment:
@asadmanzoor93 asked me for this change as it is only a one-line change.

Reply:
@asadmanzoor93 This change is not related to this PR. Please create a separate PR with a description on why we need this change.
@rehan99000 please remove this change.